core/templates/pages/story-editor-page/modal-templates/story-editor-unpublish-modal.component.html

```diff
@@ -1,70 +1,78 @@
 <div *ngIf="isSerialChapterFeatureFlagEnabled()">
   <div class="modal-header">
+    <span *ngIf="unpublishedChapters.length === 1">Unpublishing Chapter</span>
```

Comment:
Please read [this comment](https://github.com/oppia/oppia/pull/18740#discussion_r1284414121) carefully, this doesn't match what it says.

Reply:
Done